### DISCUSSION TITLE
Como es posible que cuando desaparecieron los tokens IOTA de Firefly, también desaparecieron los de Shimmer acuñados mediante stacking

### DISCUSSION THREAD

**Discussion by JALL1962** [iotaledger/firefly#8230] [Help & Support]
Cuando desaparecieron los tokens de IOTA de la wallet de Firefly, debido a la estafa de IOTALOTTERY; también desaparecieron los tokens Shimmer de la wallet de Firefly Shimmer; ¿ como es posible?

**Comment by begonaalvarezd** [comment]
El scammer tiene acceso a tus fondos en todas las redes donde usaste la misma key, si compartiste la misma key (eg mnemonic) en la shimmer network, entonces el scammer puede robar ambas redes

**Comment by andreuenigma** [comment]
¿Alguna solución por parte de IOTA?_ Al fin y al cabo el scam apareció dentro de su wallet, éso indica una vulnerabilidad por parte de la misma..., ¿o un intento de sustracción por parte de IOTA?
Yo también he sido víctima del Scam.
